src/bundle/ui-dev/src/modules/universal-discovery/content.meta.preview.module.js

```diff
@@ -94,50 +94,73 @@ const ContentMetaPreview = () => {
             </div>
         );
     };
-    const lastModifiedLabel = Translator.trans(/*@Desc("Last modified")*/ 'meta_preview.last_modified', {}, 'universal_discovery_widget');
-    const creationDateLabel = Translator.trans(/*@Desc("Created")*/ 'meta_preview.creation_date', {}, 'universal_discovery_widget');
-    const translationsLabel = Translator.trans(/*@Desc("Translations")*/ 'meta_preview.translations', {}, 'universal_discovery_widget');
+    const renderMetaPreviewLoadingSpinner = () => {
+        if (locationData && locationData.location && locationData.version) {
```

Comment:
I'm not sure if this check is sufficient. If nothing is selected and there is no loading the spinner will be visible?